test/EndToEnd/tests/InstallPackageTest.ps1

```diff
@@ -2338,7 +2338,7 @@ function Test-InstallFailCleansUpSatellitePackageFiles
     Assert-NoSolutionPackage A.fr -Version 1.0.0
 }
 
-function Test-FileTransformWorksOnDependentFile
```

Comment:
why are you disabling this test?

Reply:
because everytime you update VS on build agent, this test will fail and require manually clicking on a dialog box to proceed.

Reply:
But don't we want that scenario tested? :)